website/docs/install/install-with-script.md

```diff
@@ -6,34 +6,97 @@ sidebar_label: Quickstart
 
 import Tabs from '@theme/Tabs';
```

Comment:
I reverted to `Quickstart - Run an Ethereum Node and Stake ETH using Prysm` for SEO purposes - I want people to find this quickstart when they search for "Quickstart Prysm", "Stake Prysm" , etc. From a readability perspective I want to keep it concise, but I'm hypothesizing that verbose title / metadata will help with discoverability via search.